lib/build.gradle

```diff
@@ -36,6 +36,11 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            // https://github.com/robolectric/robolectric/issues/5115
+            all {
+                systemProperty("javax.net.ssl.trustStoreType", "JKS")
+                maxHeapSize = "1024m"
```

Comment:
these are independent of the JVM options passed in the command line (see CI script)